### PR TITLE
Remove some extra spaces in docstrings/comments.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/patches.py [hunk 1]
```diff
@@ -2562,7 +2562,7 @@ def get_y(self):
         return self._y
 
     def get_width(self):
-        "Return the width of the  rectangle"
+        "Return the width of the rectangle"
```
Comment (afvincent) [inline lib/matplotlib/patches.py]: Add a final period ? As you seem to have added it to other docstrings (at least one) :smiling_imp:.

**Comment by jklymak** [comment]
Merging because I don't think any code was touched...

File: lib/matplotlib/tri/triinterpolate.py [hunk 2]
```diff
@@ -1138,7 +1137,7 @@ def compute_geom_weights(self):
             # In the below formula we could take modulo 2. but
             # modulo 1. is safer regarding round-off errors (flat triangles).
             angle = np.abs(np.mod((alpha2-alpha1) / np.pi, 1.))
-            # Weight proportional to angle up np.pi/2 ; null weight for
+            # Weight proportional to angle up np.pi/2; null weight for
             # degenerated cases 0. and np.pi (Note that `angle` is normalized
```
Comment (afvincent) [inline lib/matplotlib/tri/triinterpolate.py]: "degenerated cases 0 and np.pi (note that `angle` is normalized"?

**Comment by afvincent** [comment]
@jklymak No problem :smile:. (BTW, I would be interested to know how all those extra spaces were found ^^...)

**Comment by anntzer** [comment]
I noticed one while checking the signature of cbook.deprecated (for #11590).  Then it just a matter of grepping for `\w \.?;? \w`...

**Pull request merged**